plugins/org.eclipse.sirius.doc/doc/user/diagrams/Diagrams.textile

```diff
@@ -738,6 +738,12 @@ The result is the same as applying _Paste Format_ and _Paste Style_.
 
 	!images/diagram_editor_paste_format2.png!
 
+h3(#paste_image). Paste image from clipboard
+
+It is also possible to paste an image from the clipboard to a node or container. To do this, copy image in your clipboard, select a node and click on button "Paste image from clipboard" (see picture below). The image is saved as a PNG file in the workspace, in an "images" folder and the selected element has a workspace image style that references this image.
```

Comment:
To do this, copy image in your clipboard => To do this, copy **the** image in your clipboard, select a node and click on **the** button "Paste image from clipboard"